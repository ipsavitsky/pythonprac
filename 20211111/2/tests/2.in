(0, 0), (0, 500), (0, 123456)
(123, 456), (321, 654), (0, 0)