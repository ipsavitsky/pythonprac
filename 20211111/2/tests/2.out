Not a triangle
32967.00
